arc/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .interface import *
```

Comment:
are IDEs cool with this? idk I've never done it

Reply:
yup; you can use it like `from arc import Riddle`, after you installed the pkg

Reply:
yes, I see that that is possible, but the star import would also make all other symbols available, such as `fg` and `bg` and whatever else is in `interface`. For clarity, we could explicitly expose what we want as a public API.

Reply:
basically, the control via `__all__.extend(whatever.__all__)` is to makes it more intuitively to see from within a file what is being exported to the `__init__`; namely, say opening `interface.py`, you can control which classes you want out, and clearly state them at the beginning of the file. the other way would be to use the `__init__` directly as an interface for this. IMO both are ok. and btw, even if you `*` import you won't be able to use `fg`, because it's not in the `__all__`.